### PR TITLE
feat(fe): redesign management edit button

### PR DESCRIPTION
**Pull request by Kimhyojung0810**
### Description

Management Problem Edit 버튼 이름 변경(Submit → Edit), 간격 조절
스크린샷 2024-08-19 오후 3 51 12

Closes TAS-707
Closes TAS-708

---

### Before submitting the PR, please make sure you do the following

- [o] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [o] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [o] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: apps/frontend/app/admin/problem/[id]/edit/page.tsx [hunk 1]
```diff
@@ -238,10 +238,10 @@ export default function Page({ params }: { params: { id: string } }) {
 
             <Button
               type="submit"
-              className="flex h-[36px] w-[100px] items-center gap-2 px-0"
+              className="flex h-[36px] w-[100px] items-center gap-1 px-0"
             >
-              <IoMdCheckmarkCircleOutline fontSize={20} />
-              <div className="mb-[2px] text-base">Submit</div>
+              <IoIosCheckmarkCircle fontSize={20} />
+              <div className="ml-1 text-base">Edit</div>
```
Comment (B0XERCAT) [inline apps/frontend/app/admin/problem/[id]/edit/page.tsx:244]: Button에 flex랑 gap이 있어서 ml은 없어도 될 것 같아요!

Comment (Kimhyojung0810) [reply apps/frontend/app/admin/problem/[id]/edit/page.tsx]: 짱이다. !!!! 🐈🐈

**Review by youznn — COMMENTED** [review]
KSJ 👍

**Comment by Kimhyojung0810** [comment]
Problem, contest 부분 edit 동일하게 수정했습니다!

**Comment by Kimhyojung0810** [comment]
버튼 크기 조절하였습니다! 100px -> 90px

**Review by youznn — APPROVED** [review]

**Pull request merged**